### DISCUSSION TITLE
解析结果被污染

### DISCUSSION THREAD

**Discussion by biliblihuorong** [kkkgo/PaoPaoDNS#169] [问答和帮助]
paopaodns 的下游是adguard home，才是给局域网解析返回记录的，同属一个ippaopaodns 为 5533 端口，adghome 为53端口，解析结果出现污染。请问 paopaodns 能查询解析日志吗？我配置了mosdns转发器，但还是没有日志记录。
![image](https://github.com/user-attachments/assets/6bf1c986-a5bb-4e87-94bd-b98fa1a1994e)
![image](https://github.com/user-attachments/assets/d73e5047-39c3-44b3-a64f-0b26a8b6ce1f)
![image](https://github.com/user-attachments/assets/27458ad5-7dcb-4dfb-abdd-359c60815159)

**Comment by kkkgo** [comment]
你可以开启ADDINFO用dig命令查询结果来源

**Reply by kkkgo** [reply]
另外建议跑一下test.sh看看dns是否工作正常。建议开启USE_MARK_DATA。

**Reply by biliblihuorong** [reply]
感谢大佬回复，后续直接使用 dig 直接对 paopaodns 进行查询结果是正确的，目前无法复现了。
test.sh ：
*********************************************************************************

images build time : 2024-07-09 17:01:09 UTC
check for the latest version ,
go to https://github.com/kkkgo/PaoPaoDNS/discussions
-> test start `1720970110`
```rust
[INFO] ALL TEST PASS.✅
```
-> test end `1720970115`

*********************************************************************************
